src/components/TodoItem.tsx

```diff
@@ -0,0 +1,62 @@
+import React, { useContext } from 'react';
+import cn from 'classnames';
+import { Todo } from '../types/Todo';
+import { TodosContext } from './TodoContext';
+
+type Props = {
+  todo: Todo;
+};
+
+export const TodoItem: React.FC<Props> = ({ todo }) => {
+  const { completed, title, id } = todo;
+
+  const todosContext = useContext(TodosContext);
+
+  if (!todosContext) {
+    return null;
+  }
+
+  const { removeTodo, isLoading } = todosContext;
+
+  function handleDeleteTodo(todoId: number) {
+    removeTodo(todoId);
+  }
+
+  return (
+    <div className={cn('todo', { completed })}>
+      <div className={cn('modal overlay', { 'is-active': isLoading })}>
+        <div className="modal-background has-background-white-ter" />
+        <div className="loader" />
+      </div>
```

Comment:
When I'm trying to delete one todo, the loader works a bit incorrectly. 
it should be shown only on the selected todo, at the moment all todos have it.  
you can try something like that
```
<div className={classNames('modal overlay', {
        'is-active': isLoading && isSelected,
      })}
      >
```